frontend/app/components/common/remote/remote-field-updater.directive.ts

```diff
@@ -88,10 +88,12 @@ function remoteFieldUpdater($http:ng.IHttpService) {
       input.on('keyup change', _.throttle(function(event:any) {
         // This prevents an update of the result list when
         // tabbing to the result list (9),
+        // pressing enter (13)
         // tabbing back with shift (16) and
         // special cases where the tab code is not correctly recognized (undefined).
         // Thus the focus is kept on the first element of the result list.
-        if (event.keyCode != 9 && event.keyCode != 16 && event.keyCode != undefined) {
+        let keyCodes = [9, 13, 16]
```

Comment:
you can use the TypeScript helper `[keyCodes.TAB, keyCodes.ENTER, keyCodes.LEFT_ARROW]` here instead to make it clear which codes are being used.